meta-lmp-bsp/recipes-bsp/u-boot/u-boot-fio/sun8i/lmp.cfg

```diff
@@ -1,8 +1,7 @@
-CONFIG_FIT=y
-CONFIG_FIT_VERBOSE=y
 CONFIG_BOOTCOUNT_LIMIT=y
 CONFIG_BOOTCOUNT_ENV=y
 CONFIG_BOOTCOUNT_BOOTLIMIT=3
-CONFIG_BOOTDELAY=-2
-CONFIG_USE_BOOTCOMMAND=y
 CONFIG_BOOTCOMMAND="fatload mmc 0:1 ${scriptaddr} /boot.itb; source ${scriptaddr}; env default -a; saveenv; reset"
+# CONFIG_FIT_SIGNATURE_STRICT is not set
+CONFIG_BOOTDELAY=2
+CONFIG_DISTRO_DEFAULTS=y
```

Comment:
We can potentially have these 3 options in another fragment (lmp-common-no-security or similar), and have it included via a conditional (UBOOT_SIGN_ENABLE basically).

Reply:
done, thanks!